.pylintrc

```diff
@@ -60,6 +60,7 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=import-star-module-level,old-octal-literal,oct-method,print-statement,unpacking-in-except,parameter-unpacking,backtick,old-raise-syntax,old-ne-operator,long-suffix,dict-view-method,dict-iter-method,metaclass-assignment,next-method-called,raising-string,indexing-exception,raw_input-builtin,long-builtin,file-builtin,execfile-builtin,coerce-builtin,cmp-builtin,buffer-builtin,basestring-builtin,apply-builtin,filter-builtin-not-iterating,using-cmp-argument,useless-suppression,range-builtin-not-iterating,suppressed-message,no-absolute-import,old-division,cmp-method,reload-builtin,zip-builtin-not-iterating,intern-builtin,unichr-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,input-builtin,round-builtin,hex-method,nonzero-method,map-builtin-not-iterating
+disable=import-error
```

Comment:
Line break didn't work. when i use line break i got this
```
Traceback (most recent call last):
  File "/usr/bin/pylint", line 3, in <module>
    run_pylint()
  File "/usr/lib/python2.7/dist-packages/pylint/__init__.py", line 23, in run_pylint
    Run(sys.argv[1:])
  File "/usr/lib/python2.7/dist-packages/pylint/lint.py", line 1269, in __init__
    linter.read_config_file()
  File "/usr/lib/python2.7/dist-packages/pylint/config.py", line 615, in read_config_file
    parser.read([config_file])
  File "/usr/lib/python2.7/ConfigParser.py", line 305, in read
    self._read(fp, filename)
  File "/usr/lib/python2.7/ConfigParser.py", line 546, in _read
    raise e
ConfigParser.ParsingError: File contains parsing errors: .pylintrc
	[line 63]: ',import-error\n'

```
yeah multiple disable worked
without disabling `import-error`
```
C:  1, 0: Missing module docstring (missing-docstring)
E:  2, 0: Unable to import 'pypandoc' (import-error)
C: 21, 8: Invalid variable name "pf" (invalid-name)
C: 23,12: Invalid variable name "pf" (invalid-name)
W: 24,14: Access to a protected member _get_pandoc_urls of a client class (protected-access)
C: 29, 0: Missing function docstring (missing-docstring)
C:  3, 0: standard import "import os" comes before "import pypandoc" (wrong-import-order)

```
by diabling `import-error`
```
C:  1, 0: Missing module docstring (missing-docstring)
C: 21, 8: Invalid variable name "pf" (invalid-name)
C: 23,12: Invalid variable name "pf" (invalid-name)
W: 24,14: Access to a protected member _get_pandoc_urls of a client class (protected-access)
C: 29, 0: Missing function docstring (missing-docstring)
C:  3, 0: standard import "import os" comes before "import pypandoc" (wrong-import-order)

```